src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Checkbox/Checkbox.jsx

```diff
@@ -1,30 +1,35 @@
 import React from 'react';
-import {compose} from 'react-apollo';
-import {connect} from 'formik';
 import * as PropTypes from 'prop-types';
 import {Toggle} from '@jahia/design-system-kit';
 import {FieldPropTypes} from '../../../../../../../../FormDefinitions/FormData.proptypes';
```

Comment:
Ce n'est pas dans la PR, mais quitte a voir ça autant le changer.
```suggestion
import {FieldPropTypes} from '~/EditPanelContainer/FormDefinitions/FormData.proptypes';
```